readme.rst

```diff
@@ -35,9 +35,7 @@ for interacting with numpy. For example (from tests/test.py)
     import pcl
     import numpy as np
     p = pcl.PointCloud(np.array([[1, 2, 3], [3, 4, 5]], dtype=np.float32))
-    seg = p.make_segmenter()
-    seg.set_model_type(pcl.SACMODEL_PLANE)
-    seg.set_method_type(pcl.SAC_RANSAC)
+    seg = pcl.Segmentation(p, model='plane', method='ransac')
```

Comment:
Is this really an improvment? using string as identifiers is bug prone and deacreses readability.
Would be better to put the model names in an enum og simply a class